src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -249,9 +334,9 @@ private static NameNode GenerateInputTypeName(string typeName)
                 inputs,
                 objectTypeDefinitionNode,
                 name,
```

Comment:
add named parameters, because two lines in a row you use `name` so it's unclear now what the value is used for.